desktop/renderer/remote-component.desktop.js

```diff
@@ -44,7 +45,7 @@ export default class RemoteComponent extends Component {
 
     this.remoteWindow.on('needProps', () => {
       try {
-        this.remoteWindow.emit('hasProps', {...this.props})
+        this.remoteWindow.emit('hasProps', {...this.props, getEngine: () => engine})
```

Comment:
trying to pass the main engine here, calling getEngin() below returns undefined...